test/client/desktop/KeyStoreFacadeTest.ts

```diff
@@ -17,174 +18,160 @@ o.spec("KeyStoreFacade test", function () {
 	let cryptoFacadeSpy: DesktopCryptoFacade
 
 	o.beforeEach(function () {
-		cryptoFacadeSpy = spyify(downcast({}))
+		cryptoFacadeSpy = spyify(downcast({generateDeviceKey: () => new Uint8Array([0, 0])}))
```

Comment:
try to us `as CryptoFacade` as it will at least check that the shape is correct

Reply:
It would be `DesktopCryptoFacade`, but the downcast saves a bunch of boilerplate because we only need one function and there is a bunch more in that interface

Reply:
It will let you cast it even if things are missing, just will check there's nothing incompatible